### PR TITLE
Correctly show list overflow error

### PR DESCRIPTION
**Pull request by MadeByMike**
Show a more helpful error message when the list page overflows the number of available items.

image
Fixes: #2063

### CODE REVIEW

File: packages/app-admin-ui/client/components/NoResults.js [hunk 1]
```diff
@@ -31,6 +31,8 @@ export const NoResults = ({ currentPage, filters, list, search }) => {
   const { onChange } = useListPagination(list.key);
   const onResetPage = () => onChange(1);
 
+  console.log(currentPage);
```
Comment (timleslie) [inline packages/app-admin-ui/client/components/NoResults.js]: Rogue console log

Comment (MadeByMike) [inline packages/app-admin-ui/client/components/NoResults.js]:
```suggestion
```

Comment (MadeByMike) [reply packages/app-admin-ui/client/components/NoResults.js]: 😱

File: packages/app-admin-ui/client/components/NoResults.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ export const NoResults = ({ currentPage, filters, list, search }) => {
   const { onChange } = useListPagination(list.key);
   const onResetPage = () => onChange(1);
 
+
```
Comment (MadeByMike) [inline packages/app-admin-ui/client/components/NoResults.js]:
```suggestion
```

File: packages/app-admin-ui/client/components/NoResults.js [hunk 3]
```diff
@@ -42,16 +42,19 @@ export const NoResults = ({ currentPage, filters, list, search }) => {
...
+    return (
       <NoResultsWrapper>
         No {list.plural.toLowerCase()} found matching the{' '}
         {filters.length > 1 ? 'filters' : 'filter'}
       </NoResultsWrapper>
     );
   }
+
   if (search && search.length) {
     return currentPage !== 1 ? (
```
Comment (MadeByMike) [inline packages/app-admin-ui/client/components/NoResults.js]: Can probably remove this one as well @jesstelford ?

Comment (jesstelford) [reply packages/app-admin-ui/client/components/NoResults.js]: Yeah, looks like it 👍

**Pull request merged**